packages/compass/src/main/auto-update-manager.ts

```diff
@@ -359,7 +359,7 @@ const STATE_UPDATE: Record<
       this.maybeInterrupt();
 
       if (isDownloadForManualCheck) {
-        ipcMain?.broadcastFocused('autoupdate:update-download-in-progress');
+        ipcMain?.broadcast('autoupdate:update-download-in-progress');
```

Comment:
@addaleax do you think it's okay to just broadcast here (for the reasons mentioned in the PR description)? Maybe it should be something like `broadcastFocusedOrFirstIfNoFocused` or something? 🤔

Reply:
I think `broadcast` is fine, yeah. I used `broadcastFocused` because it kind of made sense to me and it's what the tech design specified, but I think this is okay, we're generally moving towards using tabs more and multiple windows less anyway

Reply:
Cool, thanks for taking a look! Will keep it like this then. The tech design part is definitely on me, didn't think through some corner cases there 🙈